test/custom-resource-handlers/_cloud-formation.test.ts

```diff
@@ -17,90 +17,91 @@ const physicalId = 'Physical ID!';
 const data = {};
 const reason = 'I have reasons. Don\'t ask.';
 
-jest.mock('https');
-const httpsRequest = https.request as unknown as jest.Mock<typeof https.request>;
+const httpsRequest = https.request = jest.fn().mockName('https.request');
 
 test('sends the correct response to CloudFormation', () => {
   httpsRequest.mockImplementationOnce((opts, cb) => {
-    expect(opts.headers['content-type']).toBe('');
-    expect(opts.hostname).toBe('host.domain.tld');
-    expect(opts.method).toBe('PUT');
-    expect(opts.port).toBe('123');
-    expect(opts.path).toBe('/path/to/resource?query=string');
+      expect(opts.headers['content-type']).toBe('');
+      expect(opts.hostname).toBe('host.domain.tld');
+      expect(opts.method).toBe('PUT');
+      expect(opts.port).toBe('123');
+      expect(opts.path).toBe('/path/to/resource?query=string');
```

Comment:
I think our convention is to indent by 2-spaces, did you accidentally format the document as 4?

Reply:
Whoa I don't think I intended that...